demos/tests/run_tests.py

```diff
@@ -149,11 +149,14 @@ def main():
         dl_dir = prepare_models(auto_tools_dir, args.downloader_cache_dir, args.mo, global_temp_dir, demos_to_test)
 
         num_failures = 0
+        
+        demo_build_subdir = "/lib" if os.pathsep == ":" else ""
```

Comment:
The condition here seems rather indirect; we do not choose the path based on the path separator, we choose it based on the OS. So you should check the OS (you can use `platform.system()` to do that).

Reply:
Done.